frontend/src/app/main/ui.cljs

```diff
@@ -186,3 +188,9 @@
       (ts/schedule 100 #(st/emit! (dm/show {:content "Something wrong has happened."
                                             :type :error
                                             :timeout 5000}))))))
+
```

Comment:
delete this?

Reply:
I'll do it in other MR, i want to research more on this.

Reply:
ok